db/client.js

```diff
@@ -6,6 +6,5 @@ const client = new Client(DB_URL);
 
 // database methods
 
-// export
 module.exports = client
   // db methods
```

Comment:
looks good! we could also remove this comment since we aren't putting any db methods in this file